### PR TITLE
Issue Templates: Replace Epics With Spikes

### PR DESCRIPTION
**Pull request by agarzola**
## Description

This PR replaces our _Epic_ issue template with a new _Spike_ template.

## Motivation / Context

After some deliberation in Slack ([conversation starts here](https://chromatic.slack.com/archives/C03M0U29LF6/p1656531246710219)), we decided to start using GitHub Milestones to track Epics, with the option of using Spike tickets to track any triage/research work necessary to collect sufficient info to create an epic in the first place.

## Testing Instructions / How This Has Been Tested

A proof-read should suffice.

## Screenshots

n/a

## Documentation

I agree!

### CODE REVIEW

**Review by adamzimmermann — COMMENTED** [review]
Can I do anything to help here? I see this is a draft, so I don't want to step on toes.

**Comment by agarzola** [comment]
This will stay as a draft until the strategy outlined in the message drafted in [this Slack message](https://chromatic.slack.com/archives/C03M0U29LF6/p1657629561028689?thread_ts=1656604170.407739&cid=C03M0U29LF6) is disseminated to the rest of the team. Then this will be marked for review.

File: .github/ISSUE_TEMPLATE/spike.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+assignees: ''
+
+---
+
+## Description
+
+<!-- A clear and concise description of the issue or request that requires -->
+<!-- triage before it can be worked on. -->
+<!-- The sections suggested are intended to make it easy to create a -->
+<!-- descriptive issue Change as needed! -->
```
Comment (adamzimmermann) [inline .github/ISSUE_TEMPLATE/spike.md]:
```suggestion
<!-- descriptive issue. Change as needed! -->
```

File: .github/ISSUE_TEMPLATE/spike.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Questions / Areas of Uncertainty
+
+<!-- A list of unknowns for which we need information in order to fully -->
+<!-- understand the scope of the problem. -->
+
+## Outcome
+
+<!-- To be filled-in after all research has taken place. -->
+<!-- This may be an explanation of the issue or approach to be taken, -->
+<!-- a link to a newly-created Epic milestone, a new task/bug ticket, -->
```
Comment (adamzimmermann) [inline .github/ISSUE_TEMPLATE/spike.md]:
```suggestion
<!-- a link to a newly-created milestone, a new task/bug ticket, -->
```

**Review by adamzimmermann — APPROVED** [review]
Not sure about the second suggestion, but either way thank you for taking this on! 🙏🏼

File: .github/ISSUE_TEMPLATE/spike.md (hunk 2 above)
Comment (agarzola) [reply .github/ISSUE_TEMPLATE/spike.md]: Sure, I can see how this is unnecessary. Adding it to a batch of suggestions. Thanks!

File: .github/ISSUE_TEMPLATE/spike.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+<!-- A list of unknowns for which we need information in order to fully -->
+<!-- understand the scope of the problem. -->
+
+## Outcome
+
+<!-- To be filled-in after all research has taken place. -->
+<!-- This may be an explanation of the issue or approach to be taken, -->
+<!-- a link to a newly-created milestone, a new task/bug ticket, -->
+<!-- or a combination thereof. -->
```
Comment (agarzola) [inline .github/ISSUE_TEMPLATE/spike.md]:
```suggestion
<!-- a link to a newly-created milestone, a new task/bug ticket, or a -->
<!-- combination thereof. -->
```

**Pull request merged**